### PR TITLE
Fix build on FreeBSD (amd64 and aarch64)

### PR DESCRIPTION
**Pull request by alonsobsd**
I have added games/etlegacy to FreeBSD ports tree and those are some changes I applied for build on amd64 and aarch64. Take a look at https://www.freshports.org/games/etlegacy

### CODE REVIEW

**Review by rafal1137 — CHANGES_REQUESTED** [review]
@alonsobsd I see its more about case sensitivity here
Can you add additional entry for small one ?
```cmake
"${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "amd64")
```

**Comment by alonsobsd** [comment]
> @alonsobsd I see its more about case sensitivity here Can you add additional entry for small one ?
> 
> ```cmake
> "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "amd64")
> ```

Done

**Comment by jackeri** [comment]
Work on the platform detection is already on the way on the android branch.. Wait for it to merge (maybe later this weekend).

File: src/renderercommon/tr_font.c [hunk 1]
```diff
@@ -202,7 +202,7 @@ static glyphInfo_t *RE_ConstructGlyphInfo(int imageSize, unsigned char *imageOut
 
 		rc = FT_Load_Glyph(face, index, flags);
 
-		if (rc)
```
Comment (jackeri) [inline src/renderercommon/tr_font.c:205]: This seems a bit of a strange change

**Pull request merged**